ingest-app/docs/api.md

```diff
@@ -485,6 +485,8 @@ Example output:
 
 The bulk update API is used perform the same collection update to multiple concepts in one call. Bulk update is currently for collections only, granules are not supported.
 
+When we apply bulk update on a collection, either through API or MMT, regardless of if there is actual changes, a new collection record is created with a new revision-id, a revision-date with current date, and a updated Date field with current date, for "UPDATE" Type, in MetadataDates(add it if doesn't exist). 
```

Comment:
The wording is a bit confusing. 

I would remove "either through API or MMT". Change to "regardless if there are actual collections".

I would just say that a new revision is created. Move this all to after the next paragraph where we explain the conversion to UMM-C. Say in the UMM-JSON metadata, the Metadata Date of type "UPDATE" will be set to the current date.